force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -226,9 +278,9 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Double complexOrderOfOperations(Integer a, Integer b, Integer c, Integer d, Integer e) {
         // Add parentheses around the multiplication and subtraction operations so that they are performed before division and addition
-        Double result = a * b - c / (Double) d + e;
+        Double result = ((a * b) - c) / (Double) d + e;
```

Comment:
Order of operations will have multiplication performed before subtraction, so you shouldn't need the parentheses around a * b